internal/git/gitsync_processor_test.go

```diff
@@ -33,11 +42,29 @@ func Test_watchRepo(t *testing.T) {
 			hasError: false,
 		},
 		{
-			name: "tag name as a TargetRevision",
+			name: "remote branch name as a TargetRevision",
 			repo: v1.RepositoryPath{
-				RepoUrl:        "https://github.com/go-git/go-git.git",
-				Path:           ".github",
-				TargetRevision: "v5.5.1",
+				RepoUrl:        "https://github.com/git-fixtures/basic.git",
+				Path:           "./",
+				TargetRevision: "refs/remotes/origin/branch",
+			},
+			hasError: false,
+		},
+		{
+			name: "more branch name as a TargetRevision",
```

Comment:
```suggestion
			name: "local branch name as a TargetRevision",
```

Reply:
(wasn't sure what you meant)

Reply:
Meant for more test cases for branch name as TargetRevision for resolution. But your suggestion is meaningful. Thanks!